packages/suite/src/components/suite/modals/CancelCoinjoin.tsx

```diff
@@ -48,7 +50,7 @@ export const CancelCoinjoin = ({ onClose }: CancelCoinjoinProps) => {
                     <StyledButton
                         variant="danger"
                         onClick={() => {
-                            stopCoinjoinSession(account);
+                            dispatch(stopCoinjoinSession(account?.key));
```

Comment:
nit: `?` is redundant here
```suggestion
                            dispatch(stopCoinjoinSession(account.key));
```

Reply:
76c7b1d6d177798b40a8c34a8fb43461eb3aec34